specs/protocol/derivation.md

```diff
@@ -363,29 +363,49 @@ where:
 - `is_last` is a single byte with a value of 1 if the frame is the last in the channel, 0 if there are frames in the
   channel. Any other value makes the frame invalid (it must be ignored by the rollup node).
 
-[batcher-spec]: batching.md
-
 ### Channel Format
 
-A channel is encoded as `channel_encoding`, defined as:
+[channel-format]: #channel-format
+
+A channel is encoded by applying a streaming compression algorithm to a list of batches:
 
 ```text
 rlp_batches = []
 for batch in batches:
     rlp_batches.append(batch)
-channel_encoding = compress(rlp_batches)
 ```
 
 where:
 
 - `batches` is the input, a sequence of batches byte-encoded as per the next section ("Batch Encoding")
 - `rlp_batches` is the concatenation of the RLP-encoded batches
-- `compress` is a function performing compression, using the ZLIB algorithm (as specified in [RFC-1950][rfc1950]) with
-  no dictionary
-- `channel_encoding` is the compressed version of `rlp_batches`
+
+Prior to the Fjord upgrade,
+
+```text
+channel_encoding = zlib_compress(rlp_batches)
+```
+
+where zlib_compress is the ZLIB algorithm (as specified in [RFC-1950][rfc1950]) with no dictionary.
 
 [rfc1950]: https://www.rfc-editor.org/rfc/rfc1950.html
 
+After the Fjord upgrade, there are two possible channel encoding formats. The legacy format,
```

Comment:
Note that we use a new folder layout in the specs. This should primarily go into `specs/fjord/derivation.md`. We could then here just reference it.

Ideally you could base this PR onto https://github.com/ethereum-optimism/specs/pull/146 which will be merged very soon. The `fjord/overview.md` also needs a Brotli entry.

The specs are further modularized in this direction with https://github.com/ethereum-optimism/specs/pull/145 which is probably also merged soon.

Reply:
ok, might be best then to wait for those to merge and then I can rebase if it will happen soon?

Reply:
#146 got merged, I'm checking in on #145

Reply:
#145 won't be merged soon. Please still move this content to `specs/fjord/derivation.md` and already mention in `fjord/overview.md`, since we already use this layout for Fjord.

Reply:
done